src/Strategy/Bundler.hs

```diff
@@ -58,6 +84,23 @@ instance ToJSON BundlerProject
 instance AnalyzeProject BundlerProject where
   analyzeProject _ = getDeps
 
+instance LicenseAnalyzeProject BundlerProject where
+  licenseAnalyzeProject = fmap mconcat . traverse findLicenses . bundlerGemSpec
+
+findLicenses :: (Has ReadFS sig m, Has Diagnostics sig m) => Path Abs File -> m [LicenseResult]
+findLicenses gemspecPath = do
+  assignments <- readContentsParser (readAssignments rubyLicenseValuesP) gemspecPath
+  let licenses = foldMap value . filter isLicenseKey $ assignments
+
+  -- license keys are recommended to be SPDX, but there isn't any requirement:
+  -- https://guides.rubygems.org/specification-reference/#license=
```

Comment:
Did you ever run into this case from that URL?

> You can also use a custom license file along with your gemspec and specify a LicenseRef-<idstring>, where idstring is the name of the file containing the license text.

I had no idea that that existed. I'm mostly asking for curiousity's sake — unless you saw this in the one of the gemspecs you downloaded, I think we should fix this if and when we see it in the wild.

Reply:
I never encountered that and totally missed it myself. For now I think I'll hold off on implementing it because it seems kind of arcane.